google/ggrc-core
# GGRC-6001 Remove map:person column from export/import csv template

# Issue description

'map:person' column is displayed in export/import csv template.

# Steps to test the changes

1. Open Export page
2. Look at the Mappings: 'map:person' column is displayed
3. Open Import page and download any csv template, e.g. Regulation
4. Open csv file and search 'map:person': is shown

# Solution description

Removed the map:person column from the populating dictionary for the tables mapping rules to make sure the column is not displayed in the import/exports.

# Sanity checklist




# PR Review checklist
